Lib/test/test_wsgiref.py

```diff
@@ -788,6 +788,18 @@ def flush(self):
             b"Hello, world!",
             written)
 
+    def testConnectionAbortedError(self):
+        class AbortingWriter:
+            def write(self, b):
+                raise ConnectionAbortedError()
+
+            def flush(self):
```

Comment:
`flush()` won't be called in our case, so I think we can safely drop it.

Reply:
Done!